auto_round/script/mllm.py

```diff
@@ -515,3 +515,4 @@ def lmms_eval(args):
     )
     return results
 
+
```

Comment:
add a pre-quanization checker for awq format https://github.com/intel/auto-round/pull/396